jwst/pipeline/calwebb_coron3.py

```diff
@@ -130,21 +134,6 @@ def process(self, input):
                 image.meta.wcs = psf_sub.meta.wcs
                 target_models.append(image)
 
-            # Call outlier_detection
-            target_models = self.outlier_detection(target_models)
-
-            # Create Level 2c products
-            if target_models[0].meta.cal_step.outlier_detection == 'COMPLETE':
-                err_str1 = "Creating Level 2c output with updated DQ arrays..."
-                self.log.info(err_str1)
-                lev2c_model = psf_sub.copy()
-                # Replace Level 2b product DQ array with Level 2c DQ array
-                for i in range(len(target_models)):
-                    lev2c_model.dq[i] = target_models[i].dq
-                lev2c_model.meta.cal_step.outlier_detection = 'COMPLETE'
-                self.save_model(lev2c_model, output_file=target_file,
-                                suffix='crfints', acid=acid)
-
```

Comment:
So the level-2c's for both psf and target exposures are now created within the `outlier_detection` step itself, rather than up here in the pipeline?

Reply:
At the moment, yes. This would be more in-line with other pipelines. The `psfsub` product has always been saved, as an archive product. Once JP-1308 is complete, ideally the DQ should be propagated through. If not, well, that should become a new issue.